app/javascript/components/upload/UploadWizard.js

```diff
@@ -54,12 +54,24 @@ const STEPS = [
 const MAIN_STEPS = STEPS.slice(0, 4)
 const SUPPLEMENTAL_STEPS = STEPS.slice(4)
 
+const defaultStudyState = {
+  accession: null,
+  url_safe_name: null
+}
+
+// context to pass through UploadWizard
+export const StudyContext = React.createContext(defaultStudyState)
```

Comment:
one thing to note here is that we now have a circular import chain from UploadWizard through all the intermediate components to FileDownloadControl, and back to UploadWizard.  This is generally fine, but can lead to weird instantiation timing issues.  So despite my original recommendation to do it all inline, let's move StudyContext and defaultStudyState to `upload-utils.js`

Reply:
I think it's clearer to have the default study state just be `null` -- that way it's cleaner for components that rely on this to check 'is the study loaded?'  with `if (studyState)`. (and I see you already wisely nil-safed the property accesses in the download component)

Reply:
[30f5269](https://github.com/broadinstitute/single_cell_portal_core/pull/1365/commits/30f526987b7cb411fcd90ab0c9635d3d13ed6c60)